### PR TITLE
chore(nvim-comment): replace it with Comment.nvim.

### PR DESCRIPTION
**Pull request by ayamir**
I noticed that `nvim-comment` hasn't been updated for about six months, so I decide to use `Comment.nvim` to replace it. I remove `gb` for `BufferPick` due to `<leader>fb` provide the same function and `gb` can used by `Commment.nvim` for comment block.

### CODE REVIEW

**Comment by ayamir** [comment]
I will disable its mappings and define them in `lua/keymap/init.lua` after #485 merged.

**Comment by CharlesChiuGit** [comment]
Nice! I use `Comment.nvim` in my personal config.

File: lua/modules/configs/editor/comment.lua [hunk 1]
```diff
@@ -0,0 +1,45 @@
+return function()
+	require("Comment").setup({
+		---Add a space b/w comment and the line
+		padding = true,
+		---Whether the cursor should stay at its position
+		sticky = true,
+		---Lines to be ignored while (un)comment
+		ignore = nil,
```
Comment (CharlesChiuGit) [inline lua/modules/configs/editor/comment.lua:8]: I think it's better to ignore empty lines.
with:
```lua
ignore = "^$", -- ignores empty lines
```

**Review by CharlesChiuGit — APPROVED** [review]
LGTM

**Pull request merged**